tools/osbuilder/rootfs-builder/ubuntu/Dockerfile.in

```diff
@@ -13,6 +13,7 @@ RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive \
     apt-get --no-install-recommends -y install \
     ca-certificates \
+    clang \
```

Comment:
I got the same error as you did, But in my case, `libclang-dev` added to eliminate the error. I am not sure the difference between them.

Reply:
Fixed the PR to install `libclang-dev` instead - works!